qiskit/schemas/qobj_schema.json

```diff
@@ -152,6 +153,24 @@
                     ],
                     "title": "Frame change pulse"
                 },
+                {
+                    "properties": {
+                        "name": {
+                            "enum": [
+                                "parametric_pulse"
+                            ]
+                        },
+                        "pulse_shape": {"type": "string"},
+                        "params": {"type": "object"}
```

Comment:
This is the conflicting `params` from the main comment (along with the more general at line 262).